docs/tutorials/parsec-tutorial.md

```diff
@@ -0,0 +1,582 @@
+        "desktop": "false",
+        "image_size": "12000",
+        "headless": "true",
+        "iso_checksum": "34416ff83179728d54583bf3f18d42d2",
+        "iso_checksum_type": "md5",
+        "iso_name": "ubuntu-18.04.2-server-amd64.iso",
+        "iso_url": "http://old-releases.ubuntu.com/releases/18.04.2/ubuntu-18.04.2-server-amd64.iso",
+        "locale": "en_US",
+        "preseed" : "preseed.cfg",
+        "hostname": "gem5",
+        "ssh_fullname": "gem5",
+        "ssh_password": "12345",
+        "ssh_username": "gem5",
+        "vm_cpus": "16",
+        "vm_memory": "8192",
+        "image_name": "parsec"
+  }
+
+}
+```
+
+parsec.json is our primary .json configuration file. The provisioners and variables section of this file configure the files that need to be transferred to the disk and other things like disk image's name.
+
+Next, download packer (if not already downloaded) in the disk-image folder:
+
+```
+cd disk-image/
+wget https://releases.hashicorp.com/packer/1.4.3/packer_1.4.3_linux_amd64.zip
+unzip packer_1.4.3_linux_amd64.zip
+```
+Now, to build the disk image inside the disk-image folder, run:
+
+```
+./packer validate parsec/parsec.json
+
+./packer build parsec/parsec.json
+```
+
+## Compiling the linux kernel
+
+In this tutorial, we will use the latest LTS (long term support) release of linux kernel v4.19.83 with gem5 to run NAS parallel benchmarks.
+First, get the linux kernel config file from [here](https://github.com/darchr/gem5art/blob/master/docs/linux-configs/config.4.19.83), and place it in parsec-tests folder.
+Then, we will get the linux source of version 4.19.83:
+
+```
+git clone --branch v4.19.83 --depth 1 https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git
+mv linux linux-stable
+cd linux-stable
+```
+Compile the linux kernel from its source (using already downloaded config file config.4.19.83):
+
+```
+cp ../config.4.19.83 .config
+make -j8
+cp vmlinux vmlinux-4.19.83
+```
+
+## gem5 run scripts
+
+Next, we need to add gem5 run scripts. We will do that in a folder named configs-parsec-tests.
+Get the run script named run_parsec.py from [here](https://github.com/darchr/gem5art-experiments/blob/master/gem5-configs/configs-parsec-tests/run_parsec.py), and other system configuration files from
+[here](https://github.com/darchr/gem5art/blob/master/docs/gem5-configs/configs-parsec-tests/system/).
+The run script (run_parsec.py) takes the following arguments:
+- kernel: compiled kernel to be used for simulation
+- disk: built disk image to be used for simulation
+- cpu: the cpu model to use (e.g. kvm or atomic)
+- benchmark: PARSEC workload to run (e.g. blackscholes, bodytrack, facesim, etc.)
+- num_cpus: number of parallel cpus to be simulated
+
+## Database and Celery Server
+
+If not already running/created, you can create a database using:
```

Comment:
Can we add an external pointer here instead of repeating the same thing that's in other tutorials.?